src/EventListener/ReadListener.php

```diff
@@ -52,9 +51,9 @@ public function __construct(CollectionDataProviderInterface $collectionDataProvi
      *
      * @throws NotFoundHttpException
      */
-    public function onKernelRequest(GetResponseEvent $event)
+    public function handleEvent(EventInterface $event)
```

Comment:
This is a BC break, as you change a public method definition (for example, if any project override this service using service decoration)

Reply:
Yes sure. Maybe I will add a deprecation. It's too related to Symfony right now IMO.